tox.ini

```diff
@@ -41,7 +41,7 @@ basepython = python3
 envdir={toxworkdir}/lint
 deps = -r{toxinidir}/requirements-lint.txt
 commands =
-  mypy {posargs}
+  mypy {posargs} .
```

Comment:
The `.` causes failures when `tox` is used to run `mypy` on a single file with the command `tox -e mypy -- tests/unit/test_cli.py`